app/(panel)/apply/page.tsx

```diff
@@ -0,0 +1,7 @@
+import React from "react";
+
+const Apply = () => {
```

Comment:
Define pages like this instead

`import { NextPage } from "next";

const Apply: NextPage = () => {`